modules/api/src/main/java/org/apache/fluo/api/client/TransactionBase.java

```diff
@@ -60,4 +60,26 @@
    * encoded using UTF-8.
    */
   void setWeakNotification(CharSequence row, Column col);
+
+  /**
+   * Normally when a Fluo transaction reads data and does not write to it, it will not collide with
+   * other transactions making concurrent writes. However, all reads done using this method will
```

Comment:
Could replace `However, all reads` with `When this method is called, all reads will acquire a read lock`.